commitments-p256-signed.json

```diff
@@ -0,0 +1,25 @@
+    {
```

Comment:
I don't think renaming the file is necessary, better to just keep with the original name for existing compatibility.

Reply:
Agreed, I'll change the name back to the original one